addons/mail/static/src/new/discuss/sidebar.xml

```diff
@@ -107,6 +107,7 @@
     >
         <div class="position-relative d-flex ms-4" style="width:30px;height:30px">
             <img class="w-100 h-100 rounded-circle" t-att-src="thread.imgUrl" alt="Thread Image"/>
+            <PartnerImStatus t-if="thread.type === 'chat'" className="'o-mail-category-im-status'" partner="messaging.partners[thread.chatPartnerId]"/>
```

Comment:
To bad all these `position-relative` that come from `PartnerImStatus` using `position-absolute` internally.
Right now I know why we have them, but it might become hard to see why.

Could you move `position-absolute bottom-0 end-0` from `PartnerImStatus` to `props.className` instead?
It's a bit more diff, but at least we'd see the link between `position-relative` and corresponding `position-absolute`. Also would allow reusing `PartnerImStatus` in non-absolute positioning, like how it's done in `master` for partner mentions.